Migration/NosqlMetadaAdaptoe.py

```diff
@@ -0,0 +1,100 @@
+class MetaMongodb(object):
+    """
+    This class is for getting meta data from the database
+    """
+
+    def __init__(self, mongo_session,session,table_name, field_info):
+        self.mongo_session = mongo_session
+        self.session = session
+        self.table_name = table_name
+        self.field_info = field_info
+        self.collection_name = None
+        self.metaField = None
+
+    def retrieve_metadata_mongodb(self):
+        # we are now connected to the server
+        # mongodb creates databses and collections automatically for you.
+        print("hi welcome")
+        # db = self.mongo_session.mydb
+        #print(db)
+        # viewing database
+
+
```

Comment:
Refactor needed in this file. Remove unnecessary comments. Move core logic to core adaptor.